UnityProject/Assets/Scripts/3D/FirstPersonCamera.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+public class FirstPersonCamera : MonoBehaviour
+{
+	public float mouseSensitivity = 100f;
+
+
+	private float xRotation = 0f;
+	private float yRotation = 0f;
+
+
+	private LightingSystem LightingSystem;
+
+	public void Awake()
+	{
+		LightingSystem = Camera.main.GetComponent<LightingSystem>();
+	}
+
+	void Update()
+	{
+		LightingSystem.enabled = false;
```

Comment:
does this need to be done on update?

Reply:
the cover scenarios we go in and out of your body